PsevdoTwwiterApp/src/main/java/entity/Post.java

```diff
@@ -0,0 +1,111 @@
+package entity;
+
+import javafx.geometry.Pos;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+public class Post {
+    private long id;
+    private String description;
+    private Date createdAt;
+    private String author;
+    private String photoLink;
+    private List<String> hashTags;
+    private List<String> likes;
+
+    public Post(long id
```

Comment:
Please take into consideration that such notation is obsolete.  Have actually never seen such in a commercial project